lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -837,7 +840,22 @@ class TaskSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
       RETURN_CHILD(m_is_escalated_sp, isEscalated, bool_type);
     case 10:
       RETURN_CHILD(m_is_enqueued_sp, isEnqueued, bool_type);
-    case 11:
+    case 11: {
+      if (!m_child_tasks_sp) {
+        const auto &tasks = m_task_info.childTasks;
+        std::string mangled_typename =
+            mangledTypenameForTasksTuple(tasks.size());
+        CompilerType tasks_tuple_type =
+            m_ts->GetTypeFromMangledTypename(ConstString(mangled_typename));
+        DataExtractor data{tasks.data(), tasks.size() * sizeof(tasks[0]),
```

Comment:
The use of `sizeof` raises an eyebrow. Wouldn't you want the size of a task on the _target_ here? (For example because of 32-bit pointers on watchOS)

Reply:
The reason I did this is because data at hand (an lldb defined struct, `AsyncTaskInfo`), is internal lldb data. It comes from the target, but at this point it's internal data, not target data. Does that sound reasonable to you? It definitely felt like an uncommon scenario as I wrote this.

Reply:
Oh, if this is a `host` data structure then this is of course fine!